nibiru/chain.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "nibiru",
+  "chain_name": "Nibiru",
```

Comment:
This chain_name is not allowed. The pretty_chain_name field is for readability, but the chain_name field is for indexing. It is immutable and isn't allowed spaced or uppercased characters.

Reply:
When you say immutable, is it immutable if we are talking about lower chain envs?

Reply:
Addressed feedback, assuming chain names for lower envs can be changed

Reply:
It's a self-imposed restriction for the Chain Registry that once a chain_name is determined for a project's chain, that it cannot be changed, because it is used as a unique identifier and is referenced externally, making changes to them very dangerous and likely to cause breakage.

Reply:
the Pretty_name value is OK to change.